testing/LCZ_UCP_lookup.csv

```diff
@@ -0,0 +1,18 @@
+,FRC_URB2D,MH_URB2D_MIN,MH_URB2D,MH_URB2D_MAX,BLDFR_URB2D,H2W
+1,0.95,25,50,75,0.5,2.5
+2,0.9,10,17.5,25,0.55,1.25
+3,0.85,3,6.5,10,0.55,1.25
+4,0.65,25,50,75,0.3,1
+5,0.7,10,17.5,25,0.3,0.5
+6,0.6,3,6.5,10,0.3,0.5
+7,0.85,4,5,6,0.75,1.5
+8,0.85,3,6.5,10,0.4,0.2
+9,0.3,3,6.5,10,0.15,0.15
+10,0.55,5,10,15,0.25,0.35
+11,0,0,0,0,0,0
+12,0,0,0,0,0,0
+13,0,0,0,0,0,0
+14,0,0,0,0,0,0
+15,0.95,0,0,0,0.05,0
+16,0,0,0,0,0,0
+17,0,0,0,0,0,0
```

Comment:
we can get rid of this and simply read `w2w/resources/LZC_UCP_lookup.csv`

Reply:
ok, will fix later ...